charts/argocd-image-updater/templates/_helpers.tpl

```diff
@@ -38,7 +38,7 @@ Common labels
 helm.sh/chart: {{ include "argocd-image-updater.chart" . }}
 {{ include "argocd-image-updater.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
```

Comment:
does it make sense to only set this label when `.Chart.AppVersion` is set?

if I override the `image.tag` instead? it shouldn't be set? or should it be set to the `image.tag` value (without SHA digest)?

Reply:
simpler question: `argo-workflows` and `argo-events` actually always set this label. shouldn't this chart do the same?

Reply:
Same here. Consistency is good. I say add it. @mkilchhofer do you agree?

Reply:
:+1:

Reply:
👍